config/runtime.exs

```diff
@@ -0,0 +1,44 @@
+import Config
+
+is_prod? = config_env() == :prod
+is_release? = not is_nil(System.get_env("RELEASE_MODE"))
+
+if is_prod? and is_release? do
+  config :tzdata, :autoupdate, :disabled
+
+  config :ex_aws,
+    dynamodb: [
+      port: "DYNAMO_PORT" |> System.get_env("8080") |> String.to_integer(),
+      scheme: System.get_env("DYNAMO_SCHEME"),
+      host: System.get_env("DYNAMO_HOST")
+    ]
+
+  config :alb_monitor,
+    ecs_metadata_uri: System.get_env("ECS_CONTAINER_METADATA_URI"),
```

Comment:
Are all these `get_env`s expected to be nil sometimes vs the `fetch_env!`s which aren't? Do they have defaults set somewhere else?

Reply:
This is probably left over from when we were migrating from Beanstalk -> ECS. I think they can be all switched to `fetch_env!`.

Reply:
Fair enough, sorry I didn't noticed this is more of a moved file than a new change.